### ISSUE TITLE
marathon

### ISSUE THREAD

**Issue by JaSei** [Capgemini/mesos-ui#60]
Hi,
  I like this UI for mesos, but I try deploy via marathon (I use your marathon.json) and nothing...

Of course I set MESOS_ENDPOINT env.
```
I1116 18:27:45.457697 135617 exec.cpp:134] Version: 0.25.0
I1116 18:27:45.461282 135631 exec.cpp:208] Executor registered on slave 5db8004a-2588-41a2-8259-59ba23810c31-S1
I1116 17:27:45.902663     1 main.cpp:181] Build: 2015-07-24 10:05:39 by root
I1116 17:27:45.902845     1 main.cpp:183] Version: 0.23.0
I1116 17:27:45.902851     1 main.cpp:186] Git tag: 0.23.0
I1116 17:27:45.902854     1 main.cpp:190] Git SHA: 4ce5475346a0abb7ef4b7ffc9836c5836d7c7a66
I1116 17:27:45.903029     1 main.cpp:204] Using 'HierarchicalDRF' allocator
I1116 17:27:45.903746     1 main.cpp:383] Starting Mesos master
I1116 17:27:45.904762     1 master.cpp:368] Master 20151116-172745-822088108-5050-1 (756e8ff45677) started on 172.17.0.49:5050
I1116 17:27:45.904777     1 master.cpp:370] Flags at startup: --allocation_interval="1secs" --allocator="HierarchicalDRF" --authenticate="false" --authenticate_slaves="false" --authenticators="crammd5" --framework_sorter="drf" --help="false" --initialize_driver_logging="true" --log_auto_initialize="true" --logbufsecs="0" --logging_level="INFO" --max_slave_ping_timeouts="5" --port="5050" --quiet="false" --recovery_slave_removal_limit="100%" --registry="in_memory" --registry_fetch_timeout="1mins" --registry_store_timeout="5secs" --registry_strict="false" --root_submissions="true" --slave_ping_timeout="15secs" --slave_reregister_timeout="10mins" --user_sorter="drf" --version="false" --webui_dir="/web-ui" --zk_session_timeout="10secs"
I1116 17:27:45.905131     1 master.cpp:417] Master allowing unauthenticated frameworks to register
I1116 17:27:45.905143     1 master.cpp:422] Master allowing unauthenticated slaves to register
I1116 17:27:45.905313     1 master.cpp:459] Using default 'crammd5' authenticator
W1116 17:27:45.905326     1 authenticator.cpp:504] No credentials provided, authentication requests will be refused.
I1116 17:27:45.905334     1 authenticator.cpp:511] Initializing server SASL
I1116 17:27:45.911619    27 master.cpp:1481] The newly elected leader is master@172.17.0.49:5050 with id 20151116-172745-822088108-5050-1
I1116 17:27:45.911646    27 master.cpp:1494] Elected as the leading master!
I1116 17:27:45.911658    27 master.cpp:1264] Recovering from registrar
I1116 17:27:45.911751    33 registrar.cpp:313] Recovering registrar
I1116 17:27:45.913300    28 registrar.cpp:346] Successfully fetched the registry (0B) in 1.479936ms
I1116 17:27:45.913645    28 registrar.cpp:445] Applied 1 operations in 133890ns; attempting to update the 'registry'
I1116 17:27:45.915288    37 registrar.cpp:490] Successfully updated the 'registry' in 1.601024ms
I1116 17:27:45.915348    37 registrar.cpp:376] Successfully recovered registrar
I1116 17:27:45.915453    19 master.cpp:1291] Recovered 0 slaves from the Registry (94B) ; allowing 10mins for slaves to re-register
```
Any ideas what is wrong?

MESOS_ENDPOINT/master/state.json
MESOS_ENDPOINT/master/state-summary
return some json;)
Thanks

**Comment by enxebre** [comment]
Hi @JaSei the current version of marathon UI is fully integrated with mesos core so as explained in the Readme you can just:
```
git clone https://github.com/Capgemini/mesos-ui.git mesos-ui
cd mesos-ui
npm install
gulp build

And run mesos as the default ui for mesos like:

./bin/mesos-master.sh --ip=127.0.0.1 --work_dir=/var/lib/mesos --log_dir=/var/lib/mesos/logs --webui_dir=/path-to/mesos-ui/build
```
or if you are running mesos on docker you can just use our image like:
docker run --net=host -e MESOS_LOG_DIR=/logs capgemini/mesos-ui:latest

You can still use a marathon deployable approach using https://github.com/Capgemini/mesos-ui/tree/0.1.1

We are planning to provide support a deployable approach for current version too.
Thanks

**Comment by wallies** [comment]
This should be solved by #63. Please read the documentation readme on how to run the standalone version
